phpunit.xml

```diff
@@ -11,16 +11,16 @@
          stopOnFailure="false"
          verbose="true"
 >
-    <coverage>
-        <include>
-            <directory suffix=".php">./src</directory>
-        </include>
-        <report>
-            <html outputDirectory="build/coverage"/>
-            <text outputFile="build/coverage.txt"/>
-            <clover outputFile="build/logs/clover.xml"/>
-        </report>
-    </coverage>
+<!--    <coverage>-->
+<!--        <include>-->
+<!--            <directory suffix=".php">src</directory>-->
+<!--        </include>-->
+<!--        <report>-->
+<!--            <html outputDirectory="build/coverage"/>-->
+<!--            <text outputFile="build/coverage.txt"/>-->
+<!--            <clover outputFile="build/logs/clover.xml"/>-->
+<!--        </report>-->
+<!--    </coverage>-->
```

Comment:
**question:** What's the thought behind this coverage comment? Is this necessary for adding PHP 8.4 support? Why would this work on the other PHP versions 🤔

Reply:
the project is not using coverage in the workflows and throws some weird errors so I decided to disable these. 
If we want to fix this, it will require extra work and maybe changing the minimum requirements for the project dependencies.